src/sonic-ctrmgrd/ctrmgr/kube_commands.py

```diff
@@ -211,6 +217,56 @@ def _download_file(server, port, insecure):
     log_debug("{} downloaded".format(KUBE_ADMIN_CONF))
 
 
+def _gen_cli_kubeconf(server, port, insecure):
```

Comment:
You may need to add or update the UT in src/sonic-ctrmgrd/tests/kube_commands_test.py